README.md

```diff
@@ -41,7 +42,7 @@ available at [http://localhost:3000/](http://localhost:3000/).
 
 #### Unit Test
 
-The unit tests are powered by [chai](http://chaijs.com/) and [mocha](http://mochajs.org/).
+The unit tests are powered by [Jest](https://facebook.github.io/jest/), [Enzyme](http://airbnb.io/enzyme/) and [Sinon.JS](http://sinonjs.org/)
```

Comment:
OK then, but I will urge to tackle the migration of the e2e tests as the next priority item, otherwise it will get tedious to maintain and write tests in 2 different frameworks.

PS: IMO you can create a card for the e2e migration and work on it once this PR is merged.